### PR TITLE
initialize Brand Component package with Avatar component

### PR DESCRIPTION
**Pull request by rogervera**
## Creates the Brand Component package

[The Brand Components package will be used for components that are more tied to brand representation and/or marketing purposes. These are components that might be more frequently subject to style updates do their nature, but are not just solely used in the main Codecademy monolith app. (e.g., a brand component might also be used in a Gatsby hosted campaign page)](https://github.com/Codecademy/client-modules/blob/86671ea7005ae43cf0fa528e8bd7775b9694fe14/packages/brand-components/README.md#component-guidelines)

Also includes the first component in the package:
#631

---

## Merging your changes

When you are ready to merge to master, use the "squash and merge" button in GitHub, and follow the [commit message guide](/README.md#commit-message-guide) to write your commit message.

### CODE REVIEW

File: packages/brand-components/README.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+## Component Guidelines
+
+1. The Brand Components package will be used for components that are more tied to brand representation and/or marketing purposes. These are components that might be more frequently subject to style updates do their nature, but are not just solely used in the main Codecademy monolith app. (e.g., a brand component might also be used in a Gatsby hosted campaign page)
+
+## When to add a component to Brand Components
+
+When considering whether to add a component to Gamut, answer these questions:
+
+1. Is this component more tied to brand representation/marketing purposes than say a [Button in the Gamut package](https://github.com/Codecademy/client-modules/tree/master/packages/gamut/src/Button).
```
Comment (JoshuaKGoldberg) [inline packages/brand-components/README.md]: I think you'll want the relative path here, so folks stay within whatever commit+editor view they're already in:

```suggestion
1. Is this component more tied to brand representation/marketing purposes than say a [Button in the Gamut package](../gamut/src/Button).
```

**Comment by JoshuaKGoldberg** [comment]
Except, you know, the build failures - looks like babel needs some configuration change? I can help if you want!

**Comment by rogervera** [comment]
@joshuakgolberg

The build failures, if I'm understanding the step correctly, could be because `/src` is empty. I can wait to merge 

https://github.com/Codecademy/client-modules/pull/631 after approval to this and bundle the changes together that's alright.

File: packages/brand-components/package.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    "build:compile": "babel ./src --out-dir ./dist --copy-files --copy-ignored --extensions \".ts,.tsx\"",
+    "build:clean": "rm -rf dist",
+    "build:types": "tsc --emitDeclarationOnly",
+    "build": "yarn build:clean && yarn build:compile && yarn build:types",
+    "prepublishOnly": "yarn build"
+  },
+  "license": "MIT",
+  "devDependencies": {
+    "copyfiles": "^2.1.1"
+  }
```
Comment (jakemhiller) [inline packages/brand-components/package.json]: looks like this can be removed

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/brand-components@0.0.2-alpha.54bacf.1458+54bacfc6<br/></pre>

**Pull request merged**